.github/workflows/live-network-tests.yaml

```diff
@@ -0,0 +1,64 @@
+name: Live Network Tests
+
+on:
+  schedule:
+    - cron: '0 6,18 * * *'   # Runs daily at 6 AM and 6 PM UTC
+  workflow_dispatch: {}         # Allow manual triggering
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      LIVE_TESTS: "true"
+      LIVE_TEST_PREPROD_HOLESKY_KEY: ${{ secrets.LIVE_TEST_PREPROD_HOLESKY_KEY }}
+      LIVE_TEST_TESTNET_HOLESKY_KEY: ${{ secrets.LIVE_TEST_TESTNET_HOLESKY_KEY }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.24'
```

Comment:
please use mise. Look at the other workflows for how its done

Reply:
done